dockerfiles/jupyter/README.ipynb

```diff
@@ -10,30 +10,59 @@
+    "MLRun is an open-source AI orchestration platform for quickly building and managing continuous GenAI/ML projects across their lifecycles. It integrates into your development and CI/CD environments and automates the delivery of your production use cases.\n",
     "MLRun significantly reduces engineering efforts, time to production, and computation resources, by breaking the silos between data, ML, software, and DevOps/MLOps teams, enabling collaboration and fast continuous improvements.\n",
     "\n",
     "All the executions in MLRun are based on serverless functions. Functions are essentially Python code that can be executed locally or on a Kubernetes cluster. MLRun functions are used to run jobs, deploy models, create pipelines, and more.\n",
     "\n",
-    "The different function runtimes take care of automatically transforming the code and spec to fully managed and elastic services over Kubernetes, which saves significant operational overhead, addresses scalability, and reduces infrastructure costs. The function parameters and capabilities are explained in more detail in [Create and use functions](https://docs.mlrun.org/en/stable/runtimes/create-and-use-functions.html#create-and-use-functions)\n",
+    "The different function runtimes automatically transform the code and spec to fully managed and elastic services over Kubernetes, which saves significant operational overhead, addresses scalability, and reduces infrastructure costs. The function parameters and capabilities are explained in more detail in [Create and use functions](https://docs.mlrun.org/en/stable/runtimes/create-and-use-functions.html#create-and-use-functions)\n",
     "\n",
-    "MLRun enables users to manage and log artifacts, which are data objects produced and/or consumed by functions, jobs, or pipelines.\n",
+    "MLRun enables users to manage and log artifacts that are data objects produced and/or consumed by functions, jobs, or pipelines.\n",
     "Each artifact includes metadata attributes such as tags, data previews, and model metrics, providing essential insights for managing machine learning (ML) and large language model (LLM) projects. These attributes help users track, review, and organize different artifacts efficiently.\n",
     "\n",
-    "**For further details check the [MLRun Documentation](https://docs.mlrun.org/en/stable/index.html) including -**\n",
+    "**For further details, check the [MLRun Documentation](https://docs.mlrun.org/en/stable/index.html), including -**\n",
     " * [Projects](https://docs.mlrun.org/en/stable/projects/project.html)\n",
     " * [Functions](https://docs.mlrun.org/en/stable/runtimes/functions.html)\n",
     " * [Artifacts](https://docs.mlrun.org/en/stable/store/artifacts.html)\n",
     " * [Quick start tutorial for machine learning](https://docs.mlrun.org/en/stable/tutorials/01-mlrun-basics.html)\n",
     " * [MLRun cheat sheet](https://docs.mlrun.org/en/stable/cheat-sheet.html#mlrun-setup)"
```

Comment:
Do you want to link to setup? Or the top of the page?
     " * [MLRun cheat sheet](https://docs.mlrun.org/en/stable/cheat-sheet.html)"

Reply:
i wanted to link to the top of the page